packages/@okta/vuepress-site/docs/api/postman/password-policy.json

```diff
@@ -70,8 +70,7 @@
-    },
+      "rawModeData": "{\n    \"type\": \"PASSWORD\",\n   \"name\": \"Test Password Policy\",\n  \"description\": \"The test policy applies to users with Okta managed passwords.\",\n    \"conditions\": {\n   \"people\": {\n    \"groups\": {\n                \"include\": [\n   \"{{groupId}}\"\n              ]\n          }\n         },\n       \"authProvider\": {\n       \"provider\": \"OKTA\"\n        }\n     },\n     \"settings\": {\n        \"password\": {\n            \"complexity\": {\n            \"minLength\": 8,\n                \"minLowerCase\": 1,\n                \"minUpperCase\": 1,\n                \"minNumber\": 1,\n                \"minSymbol\": 0,\n                \"excludeUsername\": true\n            },\n            \"age\": {\n           \"maxAgeDays\": 90,\n                \"expireWarnDays\": 5,\n             \"minAgeMinutes\": 0,\n                \"historyCount\": 10\n            },\n            \"lockout\": {\n                \"maxAttempts\": 10,\n                \"autoUnlockMinutes\": 60,\n                \"showLockoutFailures\": true\n            }\n        },\n       \"recovery\": {\n       \"factors\": {\n         \"recovery_question\": {\n           \"status\": \"ACTIVE\",\n           \"properties\": {\n             \"complexity\": {\n               \"minLength\": 4\n             }\n           }\n        },\n         \"okta_email\": {\n           \"status\": \"ACTIVE\",\n           \"properties\": {\n             \"recoveryToken\": {\n               \"tokenLifetimeMinutes\": 60\n             }\n           }\n         },\n         \"okta_sms\": {\n           \"status\": \"INACTIVE\"\n         },\n         \"okta_call\": {\n           \"status\": \"INACTIVE\"\n         }\n       }\n     },\n     \"delegation\": {\n       \"options\": {\n         \"skipUnlock\": false\n       }\n    }\n    }\n  }"    },
     {
```

Comment:
It looks like this `{` may be in the wrong place?

Reply:
I think this is just the Github parser being goofy. This file is auto generated. If it works in postman, it's all good 😄